include/CLI/Option.hpp

```diff
@@ -663,13 +663,13 @@ class Option : public OptionBase<Option> {
     callback_t get_callback() const { return callback_; }
 
     /// Get the long names
-    const std::vector<std::string> get_lnames() const { return lnames_; }
```

Comment:
This might have been intended to be a const reference  `const std::vector<std::string> &`

Reply:
Good point, fixed. Also for get_snames() and the remaining getters that return more complex structures